frontend/src/js/preview/tableUtils.ts

```diff
@@ -20,7 +15,18 @@ export function useCustomTableRenderers(queryData: GetQueryResponseDoneT) {
   );
 
   const getRenderFunction = useCallback(
-    (cellType: string): ((value: CellValue) => string) | undefined => {
+    (cellType: string): ((value: CellValue) => string) => {
+      const dateFormatter = new Intl.DateTimeFormat(navigator.language, {
+        day: "2-digit",
+        month: "2-digit",
+        year: "numeric",
+      });
+
+      const currencyFormatter = new Intl.NumberFormat(navigator.language, {
+        style: "currency",
+        currency: currencyFromSymbol(currencyConfig.unit),
+      });
```

Comment:
How about extracting these out, so we don't re-initialize the formatters on every cell render?

Reply:
The changes in Table.tsx#72 should ensure, that this method is called only once per column.

Reply:
Aah, I see 👍